packages/fxa-content-server/app/scripts/templates/settings/delete_account.mustache

```diff
@@ -77,10 +77,12 @@
       </ul>
 
       <div class="input-row password-row">
-        <!-- add the autocomplete=off attribute - the user is deleting the account, they most likely don't want the password saved. -->
-        <input type="password" class="password tooltip-above" id="password" placeholder="{{#t}}Password{{/t}}" pattern=".{8,}" required autofocus autocomplete="off" />
+        <input type="password" class="password tooltip-above" id="password" placeholder="{{#t}}Password{{/t}}" pattern=".{8,}" required autofocus />
       </div>
 
+      <!-- fake input to hide "Save Password" doorhanger -->
+      <input class="hidden" required/>
```

Comment:
This appears to work, but I'm not sure why. Are you able to expand on why this works in the comment above it?

Also, small nit: could add an additional space before the `/` to make it consistent 😉

Reply:
Hey @jodyheavener , I'm not sure if my comment match the actual behavior of the browser. But I think it's a good guess.

Reply:
What you wrote makes sense to me. At the very least we know that it's intentional, so future devs can work with it as needed.